pkg/options/options.go

```diff
@@ -24,26 +24,27 @@ import (
 	"github.com/spf13/pflag"
 )
 
-type options struct {
-	Apiserver     string
-	Kubeconfig    string
-	Help          bool
-	Port          int
-	Host          string
-	TelemetryPort int
-	TelemetryHost string
-	Collectors    CollectorSet
-	Namespaces    NamespaceList
-	Version       bool
+type Options struct {
+	Apiserver                            string
+	Kubeconfig                           string
+	Help                                 bool
+	Port                                 int
+	Host                                 string
+	TelemetryPort                        int
+	TelemetryHost                        string
+	Collectors                           CollectorSet
+	Namespaces                           NamespaceList
+	Version                              bool
+	DisablePodNoneGenericResourceMetrics bool
```

Comment:
Should this be `DisablePodNonGenericResourceMetrics`?

nit: it may be easier to think about this in the affirmative, i.e. "EnablePodSpecificResourceMetrics"

Reply:
> Should this be DisablePodNonGenericResourceMetrics?

Yes. typos. Will update it.

> it may be easier to think about this in the affirmative, i.e. "EnablePodSpecificResourceMetrics"

Either works for me. Will listen for other reviewers comments about this. :)